Objects/regions.c

```diff
@@ -482,91 +482,91 @@ PyObject* _Py_MakeImmutable(PyObject* obj)
     return obj;
 }
 
-typedef struct RegionObject RegionObject;
-typedef struct RegionMetadata RegionMetadata;
+typedef struct PyRegionObject PyRegionObject;
+typedef struct regionmetadata regionmetadata;
 
-struct RegionMetadata {
+struct regionmetadata {
     int lrc;  // Integer field for "local reference count"
     int osc;  // Integer field for "open subregion count"
     int is_open;
-    RegionMetadata* parent;
-    RegionObject* bridge;
+    regionmetadata* parent;
+    PyRegionObject* bridge;
 };
 
-struct RegionObject {
+struct PyRegionObject {
     PyObject_HEAD
-    RegionMetadata* metadata;
+    regionmetadata* metadata;
     PyObject *name;   // Optional string field for "name"
 };
 
-static void RegionMetadata_inc_lrc(RegionMetadata* data) {
+static void RegionMetadata_inc_lrc(regionmetadata* data) {
```

Comment:
I think this should be:
```suggestion
static void regionmetadata_inc_lrc(regionmetadata* data) {
```
etc, below.